Team9Project/src/com/indragie/cmput301as1/ExpenseClaimListActivity.java

```diff
@@ -63,6 +64,16 @@
 	private ListModel<ExpenseClaim> listModel;
 	
 	/**
+	 * List Model of filtered expense claim.
+	 */
+	private ListModel<ExpenseClaim> filteredListModel;
+	
+	/**
+	 * List of tags to filter expense claims.
+	 */
+	private ArrayList<Tag> filteredTagsList;
```

Comment:
This can be initialized directly instead of having to set it up in `onCreate()`. E.g.

```
private ArrayList<Tag> filteredTagsList = new ArrayList<Tag>();
```